components/VerifyPresentationResult.yml

```
@@ -19,72 +19,303 @@ components:
       properties:
         verified:
           type: boolean
-          description: Overall verification assertion of the VerifiablePresentation. This is set to `True` if no errors were detected during the verification process; otherwise, `False`. See error handling section for more guidance around errors, warnings, and validation and verification steps.
+          description: Overall verification assertion of the VerifiablePresentation. This is set to `True` if the VerifiablePresentation and all contained VerifiableCredentials passed verification; otherwise, `False`. See error handling section for more guidance around errors and warnings, and validation and verification steps.
```

"all contained" is only appropriate if all VCs within VP will be checked, i.e., if the user cannot select *some* VCs to check.
```suggestion
          description: Overall verification assertion of the VerifiablePresentation. This is set to `True` if the VerifiablePresentation and all checked VerifiableCredentials passed verification; otherwise, `False`. See error handling section for more guidance around errors and warnings, and validation and verification steps.
```